ocs_ci/utility/localstorage.py

```diff
@@ -171,6 +171,6 @@ def get_lso_channel():
     """
     channel = get_ocp_version()
     # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/2324
-    if channel == '4.5':
+    if channel == '4.5' or '4.6':
```

Comment:
I think the 4.5 is out. So maybe we can start removing those workarounds for unreleased 4.5? @clacroix12 can you maybe take care of it? I've opened issue some time back:  https://github.com/red-hat-storage/ocs-ci/issues/2390 . Thanks